ansible/playbooks/roles/compatibility_layer/tasks/add_overlay.yml

```diff
@@ -9,12 +9,16 @@
   community.general.portage:
     package: eselect-repository
     state: present
+  vars:
+    ansible_python_interpreter: "{{ gentoo_prefix_path }}/usr/bin/python3"
```

Comment:
Instead of adding this to almost all tasks, we can use `set_fact` to change the interpreter after the Prefix has been installed (i.e. in `tasks/main.yml`) like:
```
name: Set Ansible Python interpreter to the Prefix one
ansible.builtin.set_fact:
    ansible_python_interpreter: "{{ gentoo_prefix_path }}/usr/bin/python3"
```

Reply:
That's more elegant, indeed.

Reply:
Changed as suggested with https://github.com/EESSI/compatibility-layer/pull/177/commits/224ee3bbc361d0ff2ce4d2a1498b160b5c03f245